src/main/java/io/jenkins/plugins/security/scan/global/Utility.java

```diff
@@ -83,101 +84,37 @@ public static boolean isStringNullOrBlank(String str) {
         return str == null || str.isBlank() || str.equals("null");
     }
 
-    public static HttpURLConnection getHttpURLConnection(URL url, EnvVars envVars, LoggerWrapper logger) {
+    public static HttpURLConnection getHttpURLConnection(URL url, LoggerWrapper logger) {
         try {
-            String proxy = getProxy(url, envVars, logger);
-            if (proxy.equals(ApplicationConstants.NO_PROXY)) {
+            ProxyConfiguration proxyConfig = Jenkins.get().proxy;
```

Comment:
As per the requirements, users will pass `HTTP_PROXY/http_proxy` or `HTTPS_PROXY/https_proxy` (must use exact env variable name as we document these in the plugin docs) from Jenkinsfile or users will set these proxy details using these exact env variable names in Jenkins environment variables and we will use those proxy to download `bring-cli`. **Users will not set proxy through Jenkins Proxy Configuration.**

Example Jenkinsfile -
```
pipeline {
    agent any
    environment {
        HTTPS_PROXY: 'https://example.proxy.com'
    }

    stages {
    // stages & steps here
    }
}
```

That's why I believe we cannot replace the proxy related codes with Jekins proxy configuration. If we must update these then we need to confirm with PM.